### DISCUSSION TITLE
字典中不存在的字怎么识别

### DISCUSSION THREAD

**Discussion by hezhenname** [PaddlePaddle/PaddleOCR#12456] [Q&A]
请问如果ppocr_keys_v1.txt字典里面不包含的字，比如“纡”，想要识别的话，需要怎么做

**Comment by lvjianjin** [comment]
把这个字加到字典里，然后准备包含包含这个字的数据集，重新训练模型

**Comment by hezhenname** [comment]
> 把这个字加到字典里，然后准备包含包含这个字的数据集，重新训练模型

这个字典有6600+的字符，数据集要怎么准备才合理，需要准备多少张图片的数据，原本能识别的字符也需要被包括吗？因为目前可能就个别字识别错误，其他的识别准确率还是挺高的，担心训练之后原本能识别的字变得不能识别，就是需要保证原本的识别准确率不下降，同时又能识别新加的字符

**Comment by lvjianjin** [comment]
理论上就是要重新整你的字典了，你可以在原来的上面增删改，构建一个你觉得合适的字典。毕竟是重新训练，“保证原本的识别准确率不下降”只能说是个没好的愿望，这就是你自己要权衡的问题了，当然最后准确率还是得看你的数据集质量。
